iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -482,7 +484,7 @@ private static async void PingServer(object ctx)
 
                     if (Logging.IsEnabled)
                     {
-                        Logging.Info(context, $"Idle time was {idleTime}, so ping request was sent.", nameof(PingServer));
+                        Logging.Info(context, $"Idle time was {idleTime}, so ping request was sent.", nameof(PingServerAsync));
```

Comment:
Would you mind putting the `if (Logging.IsEnabled) ` prefix here?

Reply:
you mean make them into a single line? line 485 already has the check .

Reply:
My mistake. OOps!